### PR TITLE
Fix kubectl apply crds link for v1.0 upgrade

### LINKED ISSUE #2043: Upgrade doc for 1.0 contains broken link
### What happened?
The upgrade guide on
https://github.com/crossplane/crossplane/blob/release-1.0/docs/guides/upgrading-to-v1.0.md
references an url which returns 404:

https://raw.githubusercontent.com/crossplane/crossplane/release-1.0/cluster/charts/crossplane/crds

### PR DESCRIPTION
**Pull request by hasheddan**
### Description of your changes

Updates link for installing updated crossplane CRDs when upgrade to v1.0
in the upgrade guide.

Signed-off-by: hasheddan <georgedanielmangum@gmail.com>

Fixes #2043 

I have:

- [x] Read and followed Crossplane's [contribution process].
- [x] Run `make reviewable test` to ensure this PR is ready for review.

### How has this code been tested

Ran updated command successfully :+1: 

[contribution process]: https://git.io/fj2m9

### CODE REVIEW

**Comment by negz** [comment]
Whoops - my bad. I wrote this guide speculatively before the branch existed and forgot to validate it after it was created. :(

File: docs/guides/upgrading-to-v1.0.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ v0.14, run:
 
 ```console
 # Update to the latest CRDs.
-kubectl apply -f https://raw.githubusercontent.com/crossplane/crossplane/release-1.0/cluster/charts/crossplane/crds
+kubectl apply -k https://github.com/crossplane/crossplane//cluster?ref=release-1.0
```
Comment (negz) [inline docs/guides/upgrading-to-v1.0.md:18]: Is this actually what we want to apply? The idea was to only apply the CRDs.

Comment (hasheddan) [reply docs/guides/upgrading-to-v1.0.md:18]: @negz this should only apply crds https://github.com/crossplane/crossplane/blob/4e38db255343f423a21b90af2615d5fb55eb45bc/cluster/kustomization.yaml#L3

Comment (negz) [reply docs/guides/upgrading-to-v1.0.md:18]: Oh, missed the `-k`. Nice.

**Review by negz — APPROVED** [review]

**Pull request merged**